homework_tester/features/methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
This file should be in `methods_and_flow_control/string_practice.rb` (not in the homework_tester/features directory).